From 4d94d5ec410714f51ffc4cd5e7c9ded0b2f66e16 Mon Sep 17 00:00:00 2001
From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 15 Feb 2022 15:47:19 +0100
Subject: [PATCH] build: Drop catchsegv Meson dependency

This is basically mostly desirable for CI. Since that still
runs under .gitlab-ci/run-test.sh which wraps execution on
catchsegv, we presumably still have the positive effects there,
or at least until we update CI to a world where catchsegv is
gone.

Closes: https://gitlab.gnome.org/GNOME/mutter/-/issues/2120
Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2294>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 2f7e40c6bc..faa38a84fc 100644
--- a/meson.build
+++ b/meson.build
@@ -338,7 +338,6 @@ if have_tests
     ],
     exe_wrapper: [
       default_test_wrappers,
-      find_program('catchsegv'),
       find_program('xvfb-run'), '-a', '-s', '+iglx -noreset',
     ],
     timeout_multiplier: 10,
-- 
GitLab

